packages/shop-mobile-expo/app.json

```diff
@@ -0,0 +1,38 @@
+{
+	"expo": {
+		"name": "@ever-platform/shop-mobile-expo",
+		"slug": "shop-mobile-expo",
+		"description": "Ever Shop Mobile App build with expo",
+		"version": "1.0.0",
```

Comment:
@Neosoulink can we put here the same version as other apps so it will be consistent?

Reply:
@Neosoulink I did that change myself too, if it's incorrect for Expo let me know.